### DISCUSSION TITLE
If I block port 22 using ufw, can I still access cockpit and will the cockpit terminal function?

### DISCUSSION THREAD

**Discussion by emanuelegori** [cockpit-project/cockpit#21408] [Q&A] [answered]
I've installed cockpit on VPS debian12 and can now access it from a custom address, https://ck.mydomain.it.
As a beginner, I have a question:
If I block port 22 using ufw, can I still access cockpit and will the cockpit terminal function?

**Comment by martinpitt** [comment, marked as answer]
Yes, if you use the Cockpit login page to log into its host, i.e. you **don't** use the "Connect to:" functionality, you don't need SSH.

For the record, this is completely opposite to what we recommend: Install cockpit-ws  or the [ws container](https://quay.io/repository/cockpit/ws) or use the [Client flatpak](https://github.com/flathub/org.cockpit_project.CockpitClient) on an uncritical machine, and connect to your production server over SSH. SSH is *much much* better protected than cockpit's web server.
